carddrawer/src/test/java/com/meli/android/carddrawer/BasicRobolectricTest.kt

```diff
@@ -0,0 +1,12 @@
+package com.meli.android.carddrawer
+
+import android.content.Context
+import android.os.Build
+import androidx.test.core.app.ApplicationProvider
+import org.robolectric.annotation.Config
+
+@Config(sdk = [Build.VERSION_CODES.LOLLIPOP_MR1])
```

Comment:
@samantaalbanez do we need this? I was checking the Java version of this same class and did not see this.

Reply:
yes, because tests that use roboletric don't work without this configuration

Reply:
Gotcha 👌